TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -1349,33 +1406,53 @@ private class LaneEnd {
                                    Color outlineColor,
                                    float size = 0.08f,
-                                   bool underground = false) {
+                                   bool underground = false,
+                                   bool showDirection = false) {
             Bounds bounds = bezier.GetBounds();
-            // Draw black outline
-            RenderManager.instance.OverlayEffect.DrawBezier(
-                cameraInfo: cameraInfo,
-                color: outlineColor,
-                bezier: bezier,
-                size: size * 1.5f,
-                cutStart: 0,
-                cutEnd: 0,
-                minY: bounds.min.y - 0.5f,
-                maxY: bounds.max.y + 0.5f,
-                renderLimits: underground,
-                alphaBlend: false);
+            float minY = bounds.min.y - 0.5f;
+            float maxY = bounds.max.y + 0.5f;
+            if (outlineColor.a != 0) {
+                // Draw black outline
+                RenderManager.instance.OverlayEffect.DrawBezier(
+                    cameraInfo: cameraInfo,
+                    color: outlineColor,
+                    bezier: bezier,
+                    size: size * 1.5f,
+                    cutStart: 0,
+                    cutEnd: 0,
+                    minY: minY,
+                    maxY: maxY,
+                    renderLimits: underground,
+                    alphaBlend: false);
+            }
 
-            // Inside the outline draw colored bezier
-            RenderManager.instance.OverlayEffect.DrawBezier(
-                cameraInfo: cameraInfo,
-                color: color,
-                bezier: bezier,
-                size: size,
-                cutStart: 0,
-                cutEnd: 0,
-                minY: bounds.min.y - 0.5f,
-                maxY: bounds.max.y + 0.5f,
-                renderLimits: underground,
-                alphaBlend: true);
+            if (color.a != 0) {
+                // Inside the outline draw colored bezier
+                RenderManager.instance.OverlayEffect.DrawBezier(
+                    cameraInfo: cameraInfo,
+                    color: color,
+                    bezier: bezier,
+                    size: size,
+                    cutStart: 0,
+                    cutEnd: 0,
+                    minY: minY,
+                    maxY: maxY,
+                    renderLimits: underground,
+                    alphaBlend: true);
+            }
+            if (showDirection) {
+                Highlight.DrawArrowHead2(
+                    cameraInfo: cameraInfo,
+                    bezier: ref bezier,
+                    t: 2f / 3f,
+                    color: Color.blue,
+                    length: 0.75f,
+                    size: 0.5f, //0.03f,
+                    minY: minY,
+                    maxY: maxY,
+                    alphaBlend: true,
+                    renderLimits: underground);
+            }
```

Comment:
@aubergine10 you can play with this part of the code to see what you like most (Hot reload works):
DrawArrowHead() for  `>` shape
DrawArrowHead2() for triangle shape
length : determines the size of the arms
size: determines the width of the arms (the thickness of the border lines cannot be controlled, increasing width increases the filled in area inside).